nistats/first_level_model.py

```diff
@@ -743,7 +748,19 @@ def first_level_models_from_bids(
                     else:
                         run_check_list.append((img_dict['ses'],
                                               img_dict['run']))
-                else:
+
+                elif '_ses-' in img_dict['file_basename']:
+                    if img_dict['ses'] in run_check_list:
+                        raise ValueError(
+                            'More than one nifti image found for the same ses '
+                            '%s, while no additional run specification present'
+                            '. Please verify that the preproc_variant and '
+                            'space_label labels were correctly specified.' %
```

Comment:
The message is fine, but you should relate this to the BIDS specs, otherwise the user may not understand what you're talking about (the users may not be fully aware of the BIDS stuff and simply use or test some features of Nistats).